Library/Homebrew/formula.rb

```diff
@@ -980,7 +998,7 @@ def migration_needed?
   end
 
   # @private
-  def outdated_versions
+  def outdated_versions(options = {})
     @outdated_versions ||= begin
```

Comment:
What should I do about it? We can cache outdated versions in two instance variables, but I think we shouldn't give up caching.